images/scripts/services.setup

```diff
@@ -126,7 +126,7 @@ systemctl stop tomcat
 # regenerate the certs used by Candlepin; this is done so the hostname
 # of the certificate is actually this container's hostname (which actually
 # is the hostname of the system, because of --uts=host)
-~candlepin/devel/candlepin/bin/deployment/gen_certs.sh --force --trust --hostname `hostname -f`
```

Comment:
FTR, not that "obvious", I actually did mean this to run on the host, as the container often has some unpredictable/random name; I wanted this to have a sensible name "services". Perhaps `hostnamectl hostname`?

Reply:
The reason I thought it was obvious was that we explicitly go out of our way to install the `hostname` command in the container specifically so that we can call it...

Reply:
The issue with using `hostnamectl hostname` (or even `hostname` without `-f`) is that it doesn't have the fully-qualified name.... `hostname -f` seems to be trying to do a lookup, and failing...